kythe/proto/repo.proto

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package kythe.proto.repo;
+
+// Represents configuration data on how to extract a repo.
+message Config {
+  repeated ExtractionHints extraction = 1;
```

Comment:
Standard style is for repeated fields to be plural.

Reply:
Ah yea, had this singular before and never remembered to update the name.  Thanks :)